docker-compose.yml

```diff
@@ -33,6 +33,7 @@ services:
       - POSTGRES_PASSWORD=debug
       - POSTGRES_USER=debug
       - POSTGRES_DB=debug
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
SCRAM auth problemi ayrıca https://github.com/acikkaynak/deprem-yardim-backend/pull/137 altında da adreslendi. Tartışmakta fayda var. Bence bu çözüm daha efektif duruyor.

Reply:
Açıkcası development ortamı olması ve olabildiğince her ortamda (cihaz/işletim sistemi bağımsız) hızlı bir şekilde ayağa kaldırılabilmesini sağlamak için #137 'de yapıldığı üzere ARM için build vs. yöntemleri yerine bunu tercih ettim.